docs/quickstart.md

```diff
@@ -184,7 +184,7 @@ with Tensor.train():
     out = net(batch)
 
     # compute loss
-    loss = sparse_categorical_crossentropy(out, labels)
+    loss = Tensor.sparse_categorical_crossentropy(out, labels)
```

Comment:
This is intentional, you're supposed to use the implementation above, this guide is run in CI to make sure it doesn't break.

Reply:
Just removed it.